### PR TITLE
#180458083

### PR DESCRIPTION
**Pull request by SalShai**
## Feature description
Portal to upload civil ID and fetch the text using Google Vision API.  

## Solution description
Create a frontend to input civil ID in form of an image(Front and Back).  Run that two images through Vision API and fetch the text. Users could opt to not upload any civil ID Image and fill out the form manually.

## Output screenshots (optional)
If User Uploads image:
Screen Shot 2021-12-12 at 3 28 15 PM

If User opt to not upload:
Screen Shot 2021-12-12 at 3 28 30 PM

## Areas affected and ensured
Frontend and backend for applicant-docs portal.

## Is there any existing behavior change of other features due to this code change?
yes,
- the UI has changed to upload images.
- The backend used is Google Vision 

## Was this feature tested on all the browsers?
  - [x] Chrome
  - [] Safari

### CODE REVIEW

File: one_fm/templates/pages/applicant-docs.html [hunk 1]
```diff
@@ -4,101 +4,236 @@
...
+                                    </div>
                             </div>
-                            <div class="file-upload-content">
-                                <img class="file-upload-image" src="#" alt="your image" />
-                                <div class="image-title-wrap">
-                                    <button type="button" onclick="removeUpload()" class="remove-image">Remove <span
-                                            class="image-title">Uploaded Image</span></button>
-                                </div>
+
+                            -->
```
Comment (pjamsheer) [inline one_fm/templates/pages/applicant-docs.html:79]: - Remove white space 
- Remove commented codes if we dont needed

File: one_fm/templates/pages/applicant-docs.js [hunk 2]
```diff
@@ -1,85 +1,168 @@
...
+      else{
+        document.getElementById("nationality").value = "";
+      }
+      
+    }
+  }); 
 }
+function upload(){
+  civilid_check = document.getElementById("noCivilID").checked;
+  //civilid_check = document.getElementById("noPassport").checked;
```
Comment (pjamsheer) [inline one_fm/templates/pages/applicant-docs.js:117]: - Remove commented codes

**Review by pjamsheer — APPROVED** [review]

File: one_fm/templates/pages/applicant-docs.js (hunk 2 above)
Comment (SalShai) [reply one_fm/templates/pages/applicant-docs.js:117]: @pjamsheer , I needed the Commented Code for the next story. So, I kept it.

File: one_fm/templates/pages/applicant-docs.html (hunk 1 above)
Comment (SalShai) [reply one_fm/templates/pages/applicant-docs.html:79]: @pjamsheer , I have resolved this in the next story. The next story was rebased on this story, so I needed the code.

**Pull request merged**